keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -737,3 +838,331 @@ This KEP has had a long journey and many revisions. Here we capture the main alt
 1. App portability is still a concern but we believe that deterministic, unique `Bucket` and `BucketAccess` names can be generated and referenced in BRs and BARs.
 
 1. Since, presumably, all or most BACs will be known to users, there is no real "control" offered to the admin with this approach. Instead, adding _allowedNamespaces_ or similar to the BAC may help with this.
+
+### Upgrade / Downgrade Strategy
+
+No changes are required on upgrade to maintain previous behaviour.
```

Comment:
Upgrade/downgrade in this case I believe would refer to the addition or removal of the CRDs to the cluster so the API will support them. Can you talk a little about how that is intended to work?